cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -893,14 +941,18 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
                 |seeing every property on the node/relationship not just the properties that the user has `READ` privilege on.""".stripMargin)
           }
           p("The resulting role should have privileges that only allow executing the UDF `apoc.any.properties`, but with elevated execution:")
-          query("SHOW ROLE boostedFunctionExecutor PRIVILEGES", NoAssertions) {
+          query("SHOW ROLE boostedFunctionExecutor PRIVILEGES AS COMMANDS", NoAssertions) {
             p("Lists all privileges for role 'boostedFunctionExecutor'")
             resultTable()
           }
 
+          note {
+            p("The `EXECUTE BOOSTED USER DEFINED FUNCTION` privileges do not apply to built-in functions, elevated privileges would have no effect on these functions.")
```

Comment:
Or:
```suggestion
            p("The `EXECUTE BOOSTED USER DEFINED FUNCTION` privileges do not apply to built-in functions, as they have no concept of elevated privileges.")
```

Reply:
sure, something like that

Reply:
Lets see what the docs team prefer (or if they have any additional ideas )

Reply:
That suggestion works!